Partition (WBP)/Functions/VI0049D.VTI0049B.fun.yml

```diff
@@ -25,22 +23,37 @@
     Define Field(#LF_rc) Reffld(#BIFA002A) Default(OK)
     Define Field(#LF_logtablefile_name) Reffld(#BIFA256A)
     
+    Define Field(#LF_DB_Type) Type(*CHAR) Length(32)
+    Use Builtin(Get_Session_Value) With_Args(DBUT) To_Get(#LF_DB_Type #LF_rc)
+    
     * Start testing.
     Clr_List Named(#wLRet)
     #Com_Owner.mthAdd_Msg( OK *Function ('Start - ' + *Function_Desc) )
     #Com_Owner.mthGetExecutionType()
-    * Export the contents of log file into an xls file only if the log file exists
+    
+    *
+    * Tests are only to be executed when run from within the MSSQL System.
+    * Only on Windows 32 bit.
+    * The MSSQL Test is only executed when its all local, not in SuperServer mode.
+    * All the other databases are ONLY run when running as a SuperServer Function (not local and not SuperServer IO)
+    
+    If (*OSBITNESS *EQ 32)
+    * If ((#LF_Execution_Type *EQ W) *And (#LF_Local_Platform *EQ W) *And (#LF_DB_Type *EQ MSSQLS) OrIf (#LF_Execution_Type *EQ F) *And (#LF_Local_Platform *EQ W) *And (#LF_Remote_Platform *EQ W) *And (#LF_DB_Type *NE MSSQLS))
```

Comment:
I gave you the text to paste in. This is not it. Go and check the text I gave you.

Reply:
I  committed with the condition you provided. Can you please clarify if  *OSBitness check required in VI0049D?

Reply:
Fixed